app/views/returnFrequency/ChooseDates.scala.html

```diff
@@ -15,11 +15,17 @@
  *@
 
 @import models.returnFrequency.ReturnDatesModel
-@import uk.gov.hmrc.play.views.html.helpers
-@import templates.inputs.radioGroup
-@import views.html.errors.errorSummary
+@import templates.inputs.RadioGroup
+@import views.html.errors.ErrorSummary
 @import models.returnFrequency._
 @import models.auth.User
+@import views.html.MainTemplate
+@import uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+
+@this(mainTemplate: MainTemplate,
+errorSummary: ErrorSummary,
```

Comment:
Where you've been injecting into views, could you tab in the parameters for readability please? I've noticed it in a few files so worth looking through